src/features/deposit/components/DepositUIMachineProvider.tsx

```diff
@@ -56,11 +56,8 @@ export const DepositUIMachineContext: DepositUIMachineContextInterface =
 
 interface DepositUIMachineProviderProps extends PropsWithChildren {
   tokenList: SwappableToken[]
-  sendTransactionNear: (transactions: Transaction[]) => Promise<string | null>
-  sendTransactionEVM: (calldata: {
-    to?: string
-    data?: string
-  }) => Promise<string | null>
+  sendTransactionNear: (tx: Transaction["NEAR"][]) => Promise<Hash | null>
```

Comment:
Near transaction won't return `Hash`. `Hash` is `0x${string}`, near transaction hash has different structure.

Reply:
Oh, you are right